ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/customized-resource/deprovision/cucushift-installer-rehearse-vsphere-ipi-customized-resource-deprovision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-customized-resource-deprovision
+  steps:
+  - chain: gather
+  - chain: gather-core-dump
+  - ref: ipi-deprovision-vsphere-diags
+  - ref: ipi-deprovision-deprovision
+  - ref: ipi-deprovision-vsphere-dns
+  - ref: ipi-deprovision-vsphere-lb
```

Comment:
For the above steps, you can directly call `chain: cucushift-installer-rehearse-vsphere-ipi-deprovision` to replace them.